xml/gl.xml

```diff
@@ -564,11 +564,11 @@ typedef unsigned int GLhandleARB;
         <enum value="0x0401" name="GL_FRONT_RIGHT" group="ColorBuffer,DrawBufferMode,ReadBufferMode"/>
         <enum value="0x0402" name="GL_BACK_LEFT" group="ColorBuffer,DrawBufferMode,ReadBufferMode"/>
         <enum value="0x0403" name="GL_BACK_RIGHT" group="ColorBuffer,DrawBufferMode,ReadBufferMode"/>
-        <enum value="0x0404" name="GL_FRONT" group="ColorBuffer,ColorMaterialFace,CullFaceMode,DrawBufferMode,ReadBufferMode,StencilFaceDirection,MaterialFace"/>
```

Comment:
Seems like a reasonable change but could break a lot downstream. Might want to justify this.

Reply:
Do you mean the `TriangleFace`?

I initially merged [ColorMaterialFace,CullFaceMode,StencilFaceDirection => MaterialFace](https://github.com/KhronosGroup/OpenGL-Registry/pull/520/commits/55d45650bf9f4914288cc741018f6a63a7cd6d33), because `MaterialFace` was used most often.
But I agree with @NogginBops that it would be better to create a new, more relevant name, as a replacement for these groups duplicating each other.

IMO:
1. Changes in XML don't just immediately appear in bindings. I would expect there to be another layer of checks by the binding maintainer when pulling a new version of this registry. At least because besides actual intended change, there might be a data scrapper quirk, applying these changes in a weird way.
That layer of checks would allow to add something like a group type synonyms, with some kind of deprecated attribute (whatever exists in a language) and then decide when to remove them, or do something else entirely.
Importantly, this is a per binding decision, for which the target language is very relevant.
And if some info isn't needed for any theoretical new binding, if it's only useful for the "compatibility" version of some of the existing bindings - it shouldn't be stored in common XML for all bindings.

2. In the first place, the point of making high-level wrapping for groups of enums is to have diagnostics and IntelliSense suggestions in the target language. So when the group name changes, for the binding user the fix should be trivial.

Of course, it's better to minimize such changes. But I think it is justified here, because removing mess from groups should be prioritized, at least for now.

Reply:
The XML is not just used by bindings, it is used by analysis tools, validation tools, etc. Anything that could possibly break anyone needs to be justified.

Reply:
I didn't know any of those. Can you please give examples?

Reply:
ARM have an internal validation tool for debugging OpenGLES tools that depend on the groups, and Google’s ANGLE use them in some capacity - likely also validation.

Reply:
Ah, that's what pings are for, I see...